tekton/release-cheat-sheet.md

```diff
@@ -40,6 +40,10 @@ the operator repo, a terminal window and a text editor.
     ```bash
     TEKTON_VERSION=# Example: v0.21.0-1
     TEKTON_RELEASE_GIT_SHA=# SHA of the release to be released
+    TEKTON_PIPELINE_VERSION=# v0.28.0
+    TEKTON_TRIGGERS_VERSION=# v0.27.0
+    TEKTON_DASHBOARD_VERSION=# v0.21.0
+    TEKTON_RESULTS_VERSION=# v0.1.1
```

Comment:
With https://github.com/tektoncd/operator/pull/493, this could be replace by `source ./test/config.sh` 😛